docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,91 @@
-# Versioning support
-
+# Versioning Support
 There are different components that might require independent versioning and support level:
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster. However, different versions of the HO are thoroughly tested only on a limited set of OpenShift versions, and this should inform your deployment decisions.
+
+For a given build of the upstream HO (e.g. [v0.1.39](https://github.com/openshift/hypershift/releases/tag/v0.1.39)), the list of OCP versions tested for Management Cluster compatibility can be found here: `link to file in repository with this information -- ideally where it is programmatically checked and results in warnings if it is not met at runtime`.
```

Comment:
We may need to delete this line. I don't think we have this information readily available 🤔

Reply:
I'm deleting this line for now since we don't seem to have this info.